docs/user_guide.md

```diff
@@ -456,6 +456,12 @@ Each line in the skip file begins with a `-` or a `+`, followed by a path glob
 pattern. `-` means that if a file matches a pattern it should **not** be
 checked, `+` means that it should be.
 
+Absolute directory paths should start with `/`.
```

Comment:
Could you please create a list from these sentences (https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)? That would be much more readable.

Reply:
Shouldn't we mention that all paths must be absolute? Perhaps it works with relative paths too, but there are edge cases when it does not. Maybe it's just simpler for us to advertise to use absolute paths (?).